lib/zclient.c

```diff
@@ -4252,3 +4257,28 @@ int zclient_neigh_ip_decode(struct stream *s, struct zapi_neigh_ip *api)
  stream_failure:
 	return -1;
 }
+
+int tun_send_zebra_gre_request(struct zclient *client,
```

Comment:
let's call this 'zclient_send_' instead?

Reply:
yes